ui/src/experiments/components/ExperimentDates.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import { InlineUL } from '../../common/components/InlineList';
+import { hasDayMonthAndYear, hasMonthAndYear } from '../../common/utils';
+
+function getDisplayFormatForDateString(date) {
+  if (hasDayMonthAndYear(date)) {
+    return 'YYYY-MM-DD';
```

Comment:
this is I think the forth or third different date format , should we consider unifying all/some if makes sense? @StellaCh 

(literature `YYYY/MM/DD`, conferences `D MMMM YYYY`)

Reply:
yup, it is a new date format. It was in the requirements, this is why I couldn't reuse others.

Reply:
We can reuse the literature one here

Reply:
Cool I guess now it could be shared then right? somehow.

Reply:
Actually no, the date in literature is already formatted in the backend. How about the date format in conferences? that would be very easy to reuse. @StellaCh

Reply:
We need to unify them, it doesn't make much sense to have different formats. The main difference is that for seminars and conferences DD is important, so it should be in the beginning, whereas for experiments we don't care so much about the specific day, we should rather focus on the year and month

Reply:
What are the options for experiment date?

Reply:
It can be anything. We can even keep it as it is now and as it is in the mockup. It's just that currently we have different formats across collections. If you say they are different use cases and that they shouldn't be the same, we can keep them different.
For literature, we have something like ```Nov 28, 1997```, for conferences ```1 June 2020``` and experiments in the mockup and how they are currently implemented have ```1992-10-01```

Reply:
let's have maximum 2 ways of displaying dates. Therefore, we keep Nov 28, 1997 and 1 June 2020 and we reuse the 1st option for experiments (Nov 28, 1997)